loaders.py

```diff
@@ -436,7 +436,7 @@ def get_datagen(self, X, y, train_mode, loader_params):
                                    train_mode=False,
                                    image_augment=None,
                                    image_augment_with_target=None,
-                                   mask_transform=None,
+                                   mask_transform=self.mask_transform,
```

Comment:
@apyskir is this intentional? Was it causing trouble when valid_datagen was used for validation during training? Does it mean `train_mode` may be confusing?

Reply:
@jakubczakon Actually the function get_datagen() here is identical to the one defined in ImageSegmentationLoaderBasic, so I removed it, because it does not need to be overriden.
Yes, it was causing some trouble, I think during validation on epoch end.
Hmm actually to calculate `validation_sum` you need to do `mask_transform`, despite it is not train mode. `train_mode` may be a little bit confusing.